lib/Rx/Observable/TimerObservable.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Rx\Observable;
+
+use Rx\Observable;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class TimerObservable extends Observable
```

Comment:
As mentioned above, let's split this in a `Timer` and `Interval`. Should simplify the code a tiny bit. E.g. the `$period > 0` check in `subscribe()` basically shows the that this class has two separate behaviors.